sklearn/utils/estimator_checks.py

```diff
@@ -688,7 +688,10 @@ def check_transformers_unfitted(name, transformer):
     X, y = _boston_subset()
 
     transformer = clone(transformer)
-    assert_raises((AttributeError, ValueError), transformer.transform, X)
+    with assert_raises((AttributeError, ValueError), msg="The unfitted "
+                       "transformer {name} does not raise an error when "
```

Comment:
Perhaps I missed it. Does this {name} thing work??? Surely we need to format the string...